src/components/player/content/index.js

```diff
@@ -18,12 +22,29 @@ const Content = ({
   swap,
   toggleFullscreen,
 }) => {
+  const {
+    index,
+  } = useCurrentInterval(storage.tldraw);
+
   if (layout.single) return null;
 
   const isTldrawWhiteboard = storage.tldraw.length ||
                              storage.panzooms.tldraw ||
                              storage.cursor.tldraw;
 
+  let presentation = <Presentation />;
+  
+  if (isTldrawWhiteboard ) {
```

Comment:
Looks like lint doesn't get this

Reply:
You're adding an eslint dependency and yet this if statement has a loose whitespace in it

Reply:
`if (isTldrawWhiteboard ) {`
-----------------------------------^--
`if (isTldrawWhiteboard) {`

It might be a symptom of the eslint not triggering or missing some config...